.devcontainer/devcontainer.json

```diff
@@ -26,7 +26,8 @@
                 "GitHub.vscode-github-actions",
                 "GitHub.vscode-pull-request-github",
                 "jebbs.markdown-extended",
-                "Gruntfuggly.triggertaskonsave"
+                "Gruntfuggly.triggertaskonsave",
+                "eamodio.gitlens"
```

Comment:
I would prefer to not add gitlens for now. It adds a lot of features and buttons... which I fear would be distracting for a new less technical users. I prefer to keep it simpler for codespace.

Reply:
I agree, GitLens is a bit overwhelming.
(Although I received some positive feedback a co-worker after the demo about the inline git blame messages).

My preference would be for keeping it, but disabling/hiding all the unused bits.